alaoweb/2020.alaoweb.org
# Use booleans and variables for CFP data elements

This PR adds two variables and booleans to `_config.yml`. These variables control the display of the 'Call for Proposals' and 'Call for Speakers' buttons, as well as the text of each button's label.

To Test:
- Enable `c4speakersEnable`. You should see a button at the bottom of the `/speakers` page to submit a proposal to become a conference speaker.
- Enable `c4sponsorsEnable`. You should see a button at the bottom of the list of conference sponsors on the main page of the site to become a conference sponsor.

Closes #12
